site/src/pages/AuditPage/AuditLogRow/AuditLogRow.tsx

```diff
@@ -232,6 +227,19 @@ export const AuditLogRow: FC<AuditLogRowProps> = ({
 	);
 };
 
+function StatusPill({ code }: { code: number }) {
+	const isHttp = code >= 100;
```

Comment:
Actually, a better measure of http status code may be whether or not this was a connect/disconnect action. Shell errors can be 0-255, but perhaps we should just mark them as 0/1 on the API side, wdyt?

Reply:
Hm... I think it works 👍